src/nemo_safe_synthesizer/config/parameters.py

```
@@ -93,6 +96,71 @@ class SafeSynthesizerParameters(Parameters):
+
+    def __init__(self, /, **data: Any) -> None:
+        """Validate constructor input with the input's recursive unknown-field policy."""
+        self.__pydantic_validator__.validate_python(
+            data,
+            self_instance=self,
+            extra=type(self)._extra_behavior(data),
+        )
+
+    # Preserve Pydantic's normal model-validation path: this initializer only
+    # selects the extras policy for direct ``Model(...)`` construction.
+    setattr(__init__, "__pydantic_base_init__", True)
```

Ruff (0.15.21)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)